openml-h2o/src/main/java/com/feedzai/openml/h2o/ClassificationH2OModel.java

```diff
@@ -106,6 +108,14 @@
     public double[] getClassDistribution(final Instance instance) {
         final AbstractPrediction abstractPrediction = predictInstance(instance);
 
+        if (isAnomalyDetection()) {
```

Comment:
Feels a little more natural to make this class abstract and have 2 extensions to implement supervised vs anomaly detection, no?

Reply:
Agreed!